metals/src/main/scala/scala/meta/internal/rename/RenameProvider.scala

```diff
@@ -357,6 +367,16 @@ final class RenameProvider(
     )
   }
 
+  private def fileThreshold(
+      files: Int
+  ): MessageParams = {
+    val message =
+      s"""|Renamed symbol is present in over $files files.
+          |It will be renamed outside the editor in the background, 
+          |otherwise the editor might become unresponsive.""".stripMargin
```

Comment:
Suggestion (feel free to discard):

s"""|The renamed symbol appears in over $files files.
    |It will be renamed without opening the files, 
    |to prevent the editor from becoming unresponsive.""".stripMargin

Reply:
Thanks, fixed!